composer.json

```diff
@@ -9,13 +9,12 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0",
+        "php": ">=7.2.5",
```

Comment:
It's not because PHPUnit is not compatible with old codebases that the package should abandon PHP 5.x. Of course, it seems to be useless, but some people may have issues with the package if this change is not done on a feature version besides all the content of this PR is actually suppose to go to a minor release. (speaking for PHP 5.6 users while I'm not part of this team for a long time...)

It's @webmozart to decide.

Reply:
How about PHP 5.x user stay on old version ?
I don't think we well have much new feature on the future.
This PR intendend only to support PHP 8 onward

Reply:
Indeed: old releases will work just fine 👍